schema.psql

```diff
@@ -0,0 +1,43 @@
+CREATE DATABASE sdc;
+
```

Comment:
I would put some DROP TABLE IF EXISTS here for your tables to make it easier if you need to reimport your data.